holidays/countries/slovenia.py

```diff
@@ -80,6 +80,10 @@ def _populate(self, year):
         # Independence and Unity Day.
         self._add_holiday_dec_26(tr("dan samostojnosti in enotnosti"))
 
+        if year == 2023:
+            # Solidarity Day. Only in 2023.
+            self._add_holiday_aug_14(tr("dan solidarnosti"))
+
```

Comment:
For one-off holidays like this, using `special_holiday` might be better (see [Argentina](https://github.com/vacanza/python-holidays/blob/master/holidays/countries/argentina.py) and [Singapore](https://github.com/vacanza/python-holidays/blob/master/holidays/countries/singapore.py)'s implementation, for example) as well as some quick testcases.
```suggestion
    special_holidays = {
        # Solidarity Day. Only in 2023.
        2023: (AUG, 14, tr("dan solidarnosti")),
    }
```

Reply:
This is more clearer yes. Thanks!